geocodio/__init__.py,sha256=YnDSNOBSwIaoGq8of9fWuSj2B_3YcsU5Vr76Qz9UNkQ,172
geocodio/_version.py,sha256=firszvVycI0sHERn72A6dEcg9bMx0XWzEYurvQ3jvHs,71
geocodio/client.py,sha256=QSzWct8GnNx6wLPbQtkVgDgEiFF3PQQ42z7FCZq5t8E,23864
geocodio/exceptions.py,sha256=2GCEE92z1v7PsRNy_5mpTA_ORb-XD4L05n3MFpbxSNU,2677
geocodio/models.py,sha256=wkuK0geGoGJ_hej3-3D8p09crMkZjrepw5opPI0QWXo,12270
geocodio_library_python-0.2.0.dist-info/METADATA,sha256=AnyGqSGPVLuf6j-HP-ZKKlzD5T033qr-iJX9PTvkgk4,6656
geocodio_library_python-0.2.0.dist-info/WHEEL,sha256=qtCwoSJWgHk21S1Kb4ihdzI2rlJ1ZKaIurTj_ngOhyQ,87
geocodio_library_python-0.2.0.dist-info/licenses/LICENSE,sha256=cXy43FWeSEvbwi2shdVczetTRHL9ySoSv4wU6sq9b9I,1092
geocodio_library_python-0.2.0.dist-info/RECORD,,
